modules/samples/consumers/wso2-event/src/main/java/org/wso2/carbon/sample/server/TestWso2EventServer.java

```diff
@@ -119,10 +120,16 @@ public void receive(List<Event> eventList, Credentials credentials) {
 
         if (protocol.equalsIgnoreCase("binary")) {
             binaryDataReceiver = new BinaryDataReceiver(new BinaryDataReceiverConfiguration(receiverPort + 100, receiverPort), databridge);
+            try {
+                binaryDataReceiver.start();
+            }
```

Comment:
Thanks. Fixed in commit: https://github.com/thiliA/product-cep/commit/c9313f04e73aba5750c51ac8600a93216d264ccb